project-reports/2023/2023-Q3-Hyperledger-Iroha.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: default
+title: 2023 Q3 Hyperledger Iroha
+parent: 2023
+grand_parent: Project Updates
+has_children: false
+has_toc: false
+---
+
+# Project Health
+
+Currently, active maintenance of Iroha 1 has stopped, and there are no plans for major releases.
+The release `2.0.0-pre-rc.18` is officially merged and stable as of July 31th.
+
+All Iroha 2 developers are now fully onboarded and at maximum productivity.
+
+The integration of Ursa essential parts to the Iroha codebase continues.
+
+# Questions/Issues for the TSC
+
+- No indexing on [2023](https://wiki.hyperledger.org/display/TSC/2023+Project+Updates) page
```

Comment:
Could you please add more details to the question `No indexing`?

Reply:
Done, thanks!